Packages/KeyAppKit/Sources/KeyAppNetwork/Dto/JsonRpcRequestDto.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  JsonRpcRequestDto.swift
+//  p2p_wallet
+//
+//  Created by Ivan on 29.04.2022.
+//
+
+import Foundation
+
+public struct JsonRpcRequestDto<T: Encodable>: Encodable {
```

Comment:
[Opinion] I would recommend to change the naming to JSONRPCRequest. Abbreviations should be capitalized and no need to add Dto at the end

Reply:
@qwerkly I think so too